lib/core/test/unit/Cardano/Wallet/Primitive/TypesSpec.hs

```diff
@@ -96,9 +98,11 @@ spec = do
 
     describe "Buildable" $ do
         it "WalletId" $ do
-            let xprv = generateKeyFromSeed (mempty, mempty) mempty
+            let Right phr =
+                    fromText "abcdefghijklmnop" :: Either TextDecodingError (Passphrase "encryption")
+            let xprv = generateKeyFromSeed (coerce phr, coerce phr) mempty
```

Comment:
Same here, the encryption passphrase should remain empty I suppose..

Reply:
corrected